wrapper/src/main/java/software/amazon/jdbc/Driver.java

```diff
@@ -105,6 +108,18 @@ public class Driver implements java.sql.Driver {
     }
   }
 
+  private final StorageService storageService;
+  private final CoreMonitorService monitorService;
+
+  public Driver() {
+    this(CoreServicesContainer.getInstance());
+  }
+
+  public Driver(CoreServicesContainer coreServicesContainer) {
+    this.storageService = coreServicesContainer.getStorageService();
```

Comment:
We probably need to replace all references of storageService and monitorService to resolving them on fly.